Makefile.am

```diff
@@ -413,9 +413,11 @@ sssdlib_LTLIBRARIES += \
     libsss_ad.la
 endif
 
+if BUILD_ID_PROVIDER_IDP
```

Comment:
Typos in the commit message "disable -> disabled", "configire -> configure"

Reply:
fixed